src/c/delete-project-modal-content.js

```diff
@@ -21,10 +22,14 @@ const deleteProjectModalContent = {
                     _project_id: args.project.project_id
                 });
                 l = postgrest.loaderWithToken(loaderOpts);
-                l.load().then(deleteSuccess(true));
+                l.load().then(() => {
```

Comment:
@adrianob let's add the catch part here and just show a generic network error copy.